win32/Lib/pywin32_bootstrap.py

```diff
@@ -7,22 +7,27 @@
 # If Python has `os.add_dll_directory()`, we need to call it with this path.
 # Otherwise, we add this path to PATH.
 import os
-import site
 
-# The directory should be installed under site-packages.
 
-dirname = os.path.dirname
-# This is to get the "...\Lib\site-packages" directory
-# out of this file name: "...\Lib\site-packages\win32\Lib\pywin32_bootstrap.py".
-# It needs to be searched when installed in virtual environments.
-level3_up_dir = dirname(dirname(dirname(__file__)))
+try:
+    Exc = ModuleNotFoundError  # Introduced in Python 3.6
```

Comment:
Is this strictly necessary? IIUC, `ModuleNotFoundError` derives from `ImportError` and I don't see a good reason to differentiate.

Reply:
My instinct is generally to pick the most specific possible exception for `except` to match. This avoids hiding errors unrelated to the absence of `pywin32_system32`. There would be some bizarre circumstances that would lead to such hiding, but the cost of being specific here seems low to me. Anyway, I won't die on this hill. If you insist, I'll just use `ImportError`.

Reply:
I don't feel *too* strongly about it, and in general I'd agree, but in this case I feel the added complexity from changing behaviors based on the version, when in practice it's exactly the same behavior regardless, swings the argument to just catching ImportError.

Reply:
Fixed in 7a6de27